ideascube/management/commands/tags.py

```diff
@@ -121,3 +129,57 @@ def list(self, options):
         print(row.format('.' * 40, '.' * 40, '.' * 40))
         for tag in Tag.objects.order_by('slug'):
             print(row.format(tag.name, tag.slug, self._count(tag.name)))
+
+    def sanitize(self, options):
+        all_tags = ((sanitize_tag_name(t.name), t) for t in Tag.objects.all())
+        all_tags = sorted(all_tags, key=itemgetter(0))
+        all_tags = groupby(all_tags, key=itemgetter(0))
+        for new_tag_name, tags in all_tags:
+            tags = (t[1] for t in tags)
```

Comment:
Oh right, I forgot about that!

So you end up twice with the sanitized tag name in the iterator, one that was used by sorted/groupby, and one which is the result of the groupby. 😞